README.md

```diff
@@ -36,87 +40,40 @@ The following options are available:
 |  `-bl`, `--backoff_low` |  set a lower backoff bound (only for msqueue, lfstack and lfuobject benchmarks)       |
 |  `-h`, `--help`         |  displays this help and exits                                                         |
 
-In order to perform smoke test: `./run_all.sh`. This will quickly run all available benchmarks with default options
-and store the results in the `results.txt` file.
+The framework provides the `validate.sh` validation/some script. The `validate.sh` script compiles the sources in `DEBUG` mode and runs a big set of benchmarks for different amounts of threads. After running each of the benchmarks, the script evaluates the `DEBUG` output and in case of success it prints `PASS`. In case of a failure, the script simple prints `FAIL`. In order to see all the available options of the validation/smoke script, execute `validate.sh -h`. The follow image shows the execution and the default behavior of `validate.sh`.
```

Comment:
Maybe ` for different amounts of threads`  --> ` with various number of threads`

Reply:
Fixed with "with various number**s** of threads".